src/openrct2-ui/windows/Banner.cpp

```diff
@@ -176,6 +178,8 @@ static void window_banner_mouseup(rct_window* w, rct_widgetindex widgetIndex)
     int32_t y = banner->position.y << 5;
 
     TileElement* tile_element = map_get_first_element_at(x / 32, y / 32);
+    if (tile_element == nullptr)
+        return;
```

Comment:
This would technically block a window close that could be valid. Its not worth changing though. Keep like this.

Reply:
Ok, not changing anything on this one then.